src/Breadcrumb/README.md

```diff
@@ -68,18 +68,42 @@ Use as a secondary navigation pattern to help convey hierarchy and enable naviga
 }
 ```
 
+## With custom link element
+
+By default `Breadcrumb` uses `a` tag to render breadcrumbs, which may not always suit your needs. 
+This behaviour can be customized with `linkAs` prop, the example below uses Gatsby's `Link` component, but it would also work with [react-router's `Link`](https://reactrouter.com/en/main/components/link) component as well because they share required parts of the component API.
+
+Note that `links` list contains objects with different keys compared to the example above, specifically `href` key is replaced with `to`, that's because Gatsby's `Link` expects its destination to be set through `to` prop (same as react-router's `Link`), internally `Breadcrumb` passes down these objects (except `label` attribute) as props to the `linkAs` element. 
+
+```jsx live
+() => {
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+
+  return (
+    <Breadcrumb ariaLabel="Breadcrumb basic"
+      links={[
+        { label: 'Home', to: '/' },
+        { label: 'CSS Utilities', to: '/foundations/css-utilities' },
+      ]}
+      isMobile={isExtraSmall}
+      linkAs={GatsbyLink}
```

Comment:
This example uses Gatsby's `Link` but I've tested it with react-router also, didn't want to add it to project's dependencies just for the sake of one example. (fwiw Gatsby's `Link` might even be using react-router under the hood, they do seem quite similar)

Reply:
I believe Gatsby actually uses `reach-router` ([docs](https://reach.tech/router/)), which has nearly the API as `react-router`.